app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
where do you check  that the `cp ` input command is correct.? Read checklist

Reply:
Just like if  "cp" in command? or something else?

Reply:
as well you need to check if command consists of three parts